extensions/skyportal/skyportal/handlers/api/alert.py

```diff
@@ -33,41 +34,61 @@
 from skyportal.utils.calculations import great_circle_distance
 
 from ...models import (
+    Candidate,
+    Classification,
+    Filter,
     Group,
     Instrument,
     Obj,
     Source,
     Stream,
     User,
-    Candidate,
-    Filter,
-    Classification,
 )
+from ...utils.parse import str_to_bool
 from ..base import BaseHandler
 from .photometry import add_external_photometry
 from .thumbnail import post_thumbnail
 
-alert_available = True
+matplotlib.pyplot.switch_backend("Agg")
```

Comment:
Should this be OS specific?

Reply:
It's required on some platforms I believe (if you run matplotlib in some backend) so I just like to add it by default. If your on a platform that needs it it's necessary anyways if you want it to run without warnings or issues, and on platforms that don't need it it won't change much of anything if you have that option on.